src/index.js

```diff
@@ -7,9 +7,12 @@ import reportWebVitals from "./reportWebVitals";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <BrowserRouter>
-    <AppRoutes />
-  </BrowserRouter>
+  // @TODO: https://jiraent.cms.gov/browse/ISPGBSS-1209
```

Comment:
Hmm I didn't know a lot about react strict mode before your PR! But I did some reading about it. I didn't know that it would make multiple api calls and such to help detect state and memory leaks in the app! (Cool stuff) So it's nice that we can use it for development, but obviously don't want it for use in production. Thankfully, react knows this and added a helpful note at the top of their [docs about strict mode](https://reactjs.org/docs/strict-mode.html): 

> **Note:**
Strict mode checks are run in development mode only; _they do not impact the production build._

And when we're ready to deploy to production, we will need to use [a production build](https://reactjs.org/docs/optimizing-performance.html#use-the-production-build) of our app.

**tl;dr** Leaving `<React.StrictMode>` in forever should be safe and we shouldn't have to manually handle removing it for production as long as we correctly do react production builds! :)